exe/launcher/Main.hs

```diff
@@ -74,15 +81,17 @@ main = do
     when (args `isPresent` (shortOption 'h')) $ help cli
     when (args `isPresent` (longOption "version")) $ putStrLn getVersion
 
-    network <- args `parseArg` longOption "network"
+    let stateDir = args `getArg` (longOption "state-dir")
```

Comment:
It would be nice to add _integration_ test that'd check if the directory and contents are created correctly.
Could be here -> https://github.com/input-output-hk/cardano-wallet/tree/master/lib/http-bridge/test/integration/Test/Integration/Scenario/CLI